### ISSUE TITLE
CMake generator expressions not handled.

### ISSUE THREAD

**Issue by Anteru** [pygments/pygments-migration-test#649]
(Original issue 942 created by None on 2013-12-26T01:34:04.149943+00:00)

CMake has a generator-expression syntax:

http://www.cmake.org/cmake/help/git-master/manual/cmake-generator-expressions.7.html

Pygments issues an error if it is encountered. The following patch remedies that:

```
#!diff

--- text-new.py 2013-12-26 02:29:55.120153412 +0100
+++ /usr/lib/python2.7/dist-packages/pygments/lexers/text.py    2013-12-26 02:24:31.916166929 +0100
@@ -1640,6 +1640,7 @@
             (r'\(', Punctuation, '#push'),
             (r'\)', Punctuation, '#pop'),
             (r'(\${)(.+?)(})', bygroups(Operator, Name.Variable, Operator)),
+            (r'(\$<)(.+?)(>)', bygroups(Operator, Name.Variable, Operator)),
             (r'(?s)".*?"', String.Double),
             (r'\\\S+', String),
             (r'[^\)$"# \t\n]+', String),

```

**Comment by Anteru** [comment]
(Original issue 942 last updated on 2014-01-09T19:30:16.849106+00:00)

(Issue automaticaly closed due to status in Bitbucket: resolved)

**Comment by Anteru** [comment]
(Original comment by birkenfeld on 2014-01-09T19:29:36.620754+00:00)

Thanks!

**Comment by Anteru** [comment]
(Original comment by birkenfeld on 2014-01-09T19:30:16.843358+00:00)

Closes #942: add CMake generator expressions.

→ <<cset 4134237fc42e>>
